Sources/TuistGenerator/Generator/SchemesGenerator.swift

```diff
@@ -18,16 +31,40 @@ protocol SchemesGenerating {
                                 xcprojectPath: AbsolutePath,
                                 generatedProject: GeneratedProject,
                                 graph: Graphing) throws
+    
+    func wipeSchemes(at: AbsolutePath) throws
```

Comment:
I'd add documentation to the method and explain why wiping the schemes is necessary.